.github/workflows/ci.yml

```diff
@@ -0,0 +1,238 @@
+name: CI
+
+# Trigger on pull_request or on push to master
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+
+  lint:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/setup-python@v1
+      with:
+        python-version: '3.x'
+    - uses: actions/checkout@v1
+      with:
+        submodules: true
```

Comment:
do we have submodules?

Reply:
No, you are right we could remove those lines.   Its harmless though.